test/src/com/xilinx/rapidwright/design/compare/TestDesignComparator.java

```diff
@@ -149,25 +149,25 @@ public void testDesignComparator() {
         Assertions.assertTrue(net.hasPIPs());
         test2.removeNet(net);
 
-        compareDesign(22, 1, DesignDiffType.NET_MISSING, dc, gold, test2);
+        compareDesign(28, 1, DesignDiffType.NET_MISSING, dc, gold, test2);
```

Comment:
I don't think there's any `SiteInst` renaming happening in this test, so are the increased number of diffs due to iteration order (which shouldn't matter?)

Reply:
This changed because the `SiteInst` renaming was taking placed at the end of the DCP load when `ModuleInst` objects were being formed (RapidWright has added special metadata to reconstruct them after a DCP has traditionally been fully loaded).  Because of the rename taking place, one of the sites was not being considered previously.  The new code now handles the `SiteInst` that was being left out before the change.